fbpic/main.py

```diff
@@ -312,13 +326,11 @@ def step(self, N=1, correct_currents=True,
         if self.comm.size > 1 and correct_divE:
             raise ValueError('correct_divE cannot be used in multi-proc mode.')
 
-        # Let the user know that the first step is much longer
-        if show_progress and (self.comm.rank==0):
-            print('Performing %d PIC iterations:' %N )
```

Comment:
I would have a slight preference for keeping `print('Performing %d PIC iterations:' %N )` (this would then be moved to the `printing_utils.py`).
What do you think?

Reply:
I would still remove this line. It would only be printed if show_progress == True and in this case the total number of iterations are printed by the progress bar anyways. So this would be redundant.

Reply:
OK, sounds good.